src/couch_replicator/src/couch_replicator_doc_processor.erl

```diff
@@ -108,11 +112,6 @@ get_worker_ref({DbName, DocId}) when is_binary(DbName), is_binary(DocId) ->
             nil
     end.
 
-% Cluster membership change notification callback
```

Comment:
We don't need the cluster membership change notification any longer as the check_membership periodic check would take care of it. couch_replicator_db_changes will still initiate a rescan when cluster nodes connect/disconnect.